### PR TITLE
Improve build disk usage

### PR DESCRIPTION
**Pull request by telphan**
@agherzan @floion

### CODE REVIEW

**Comment by agherzan** [comment]
LGTM

**Comment by telphan** [comment]
Related to #7

File: edison.coffee [hunk 1]
```diff
@@ -4,8 +4,10 @@ deviceTypesCommon = require 'resin-device-types/common'
 OSX_DFU_COREUTILS_USBUTILS = '''
 	You need <a href="https://www.macports.org/">MacPorts</a> installed on your system.
 	<br>
-	Run the following to install <code>dfu-util</code>, <code>usbutils</code> and <code>coreutils</code>:
-	<code>sudo port install dfu-util@0.7 usbutils coreutils && sudo port activate dfu-util@0.7</code>
+	Run the following to install <code>usbutils</code> and <code>coreutils</code>:
+	<code>sudo port install usbutils coreutils</code>
+  You will also need <code>dfu-util</code> we recommand this to be installed with brew:
```
Comment (floion) [inline edison.coffee]: Add a comma here somewhere. Or maybe split the above into 2 sentences

File: layers/meta-resin-edison/conf/samples/local.conf.sample [hunk 2]
```diff
@@ -34,6 +34,9 @@ DISTRO ?= "resin-systemd"
 # http://www.yoctoproject.org/docs/latest/mega-manual/mega-manual.html#ref-classes-rm-work
 #INHERIT += "rm_work"
 
+# Remove the old image before the new one generated to save disk space
```
Comment (floion) [inline layers/meta-resin-edison/conf/samples/local.conf.sample]: Maybe you could reword the comment:

"Remove the old image before the new one generated to save disk space"

feels a bit weird. Maybe "is generated" is more appropriate

**Comment by floion** [comment]
LGTM, but previously you had also a edison.coffee commit in this PR, in which you modified the OSX_DFU_COREUTILS_USBUTILS instructions. Did you delete this commit on purpose or by mistake?

**Comment by telphan** [comment]
@floion The commit was deleted on purpose, because brew does not allow me to get a specific version of dfu-util. I am looking into alternative ways on doing that.

**Pull request merged**